applications/openshift/kubelet/kubelet_configure_event_creation_master/rule.yml

```diff
@@ -0,0 +1,69 @@
+documentation_complete: true
```

Comment:
I'm not sure how feasible this is, but it would be nice to have a single rule that accepted variables are inputs. I started thinking about this because we have variables for "worker" and "master" roles, but we hard-code the roles in the actual rule name.

There also seems to be some duplication between master and worker rules.

Reply:
the worker and master rule will not be shown in the compliance check result

Reply:
Only the main rule is used in the profile